lib/jekyll/convertible.rb

```diff
@@ -278,10 +111,8 @@ def do_layout(payload, layouts)
     def write(dest)
       path = destination(dest)
       FileUtils.mkdir_p(File.dirname(path))
-      File.open(path, 'wb') do |f|
-        f.write(output)
-      end
-      Jekyll::Hooks.trigger hook_owner, :post_write, self
+      File.write(path, output)
```

Comment:
The `wb` mode helped with content that was non-ASCII. We need to make sure we keep that mode.